Glaze/README.md

```diff
@@ -0,0 +1,21 @@
+# Glaze
+
+## Screenshots
+
+### Base
+<img src="https://github.com/CharlieS1103/Glaze-theme/blob/main/screenshots/base.png" alt="img"> 
+
+
+### Dark
+<img src="https://github.com/CharlieS1103/Glaze-theme/blob/main/screenshots/dark.png" alt="img" "> 
```

Comment:
There is a syntax error here, the image is not displayed. I wonder why you didn't just use the Markdown directive for displaying the images